Buildings/Controls/OBC/ASHRAE/PrimarySystem/BoilerPlant/Staging/Processes/Up.mo

```diff
@@ -0,0 +1,651 @@
+within Buildings.Controls.OBC.ASHRAE.PrimarySystem.BoilerPlant.Staging.Processes;
+block Up
+    "Sequence for control devices when there is stage-up command"
+
+  parameter Boolean primaryOnly = false
```

Comment:
Please change to `is_priOnl` for consistency with other flags, such as `have_WSE` or `is_parChi`. @mwetter, just to double check, for the boolean flags we should also stick to the 3 letters rule (please see the next comment in line 9 as well)?